rs/replicated_state/src/canister_state/queues.rs

```diff
@@ -1102,7 +1104,7 @@ impl CanisterQueues {
         request: Request,
         reject_context: RejectContext,
         subnet_ids: &[PrincipalId],
-    ) -> Result<(), StateError> {
+    ) -> Result<bool, StateError> {
```

Comment:
I don't think the `bool` here has any meaning. We can't not find a slot or the canister altogether, so the response here would always be `Ok(true)` or `Err(_)`.

So instead of exposing it in the return value without any need or explanation, I'd rather we swallow it below and retain the existing `Result<(), StateError>` return type.

Reply:
Changed it back to `()`. Also one level above.